api/all.yaml

```diff
@@ -1490,6 +1832,206 @@ components:
         - READ_VOLUME
         - WRITE_VOLUME
       type: string
+    RegisteredModelInfo:
+      type: object
+      properties:
+        name:
```

Comment:
Databricks hosted API has a `name` and a `full_name`, FWIW.

Reply:
(Seems redundant.)

Reply:
Adding full_name.  I think this is early tech debt that was never cleaned up (why both are there).